new/LIT7034Lateamar.xml

```diff
@@ -0,0 +1,81 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7034Lateamar" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ለትእማር፡ ደሴተ፡ ክብራ፡ ዘበድንግልና፡ ትሄሉ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">La-Tǝʾmār dasseta kǝbra za-ba-dǝngǝlǝnna tǝhellu ... (Qǝne of wāzemā type)</title>
+                <title xml:lang="en" corresp="#t1">The island of Kǝbrān that guards Tamar in chastity ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>         
+                <p>This <term key="Qene">qǝne</term> poem, attested in <ref type="mss" corresp="Tanasee6"/> 
+                    readable only in part, is of the type <foreign xml:lang="gez">wāzemā</foreign>. 
```

Comment:
This could be a place to refer to the authority file

Reply:
I think, Denis wants to use the keywords for the qene modes only in textClass to avoid double encoding that leads to double work and double results in searches.